openreview/conference/templates/areachairWebfield.js

```diff
@@ -162,17 +158,32 @@ var loadData = function(acPapers, secondaryAcPapers) {
     blindedNotesP = Webfield.getAll('/notes', {
       invitation: BLIND_SUBMISSION_ID,
       number: noteNumbersStr,
+      select: 'id,number,forum,content,details',
       details: 'invitation,replyCount'
     }).then(function(notes) {
       return _.sortBy(notes, 'number');
     });
 
-    var noteNumberRegex = noteNumbers.join('|');
-    metaReviewsP = Webfield.getAll('/notes', {
-      invitation: getInvitationRegex(OFFICIAL_META_REVIEW_NAME, noteNumberRegex)
+    metaReviewsP = _.map(noteNumbers, function(noteNumber) {
```

Comment:
doing a request per paper is better than a single one with a regex?

Reply:
Can we do here get submission with directReplies and get the meta reviews from the details object of the submission?

Reply:
Yes, I've already made the change